browser/src/map/handler/Map.Feedback.js

```diff
@@ -113,7 +113,8 @@ L.Map.Feedback = L.Handler.extend({
 			      { wsdhash : window.app.socket.WSDServer.Hash },
 			      { 'lokit_hash' : lokitHash },
 			      { 'wopi_host_id' : wopiHostId },
-			      { 'proxy_prefix_enabled' : proxyPrefixEnabled }];
+			      { 'proxy_prefix_enabled' : proxyPrefixEnabled },
+			      { 'doc_type': this._map._docLayer._docType}];
```

Comment:
`map` has `getDocType()` getter for it (always good to hide implementation details):

https://github.com/CollaboraOnline/online/blob/d4d03c1ea0eede6505f78c730ba25e4ebd9e0b40/browser/src/control/Control.SheetsBar.js#L90